lk2nd/device/dts/msm8952/msm8937-nokia-nd1.dts

```diff
@@ -4,37 +4,41 @@
 #include <lk2nd.dtsi>
 
 / {
-	qcom,msm-id = <QCOM_ID_MSM8937 0x2000>;
+	qcom,msm-id = 	<QCOM_ID_MSM8937 0x0>,    // Variant: FIH_D1_EVB
+			<QCOM_ID_MSM8937 0x1000>, // Variant: FIH_D1_EVT
```

Comment:
In this case if they're clearly labeled as EVB and EVT (and for ple, also DVT since there is PVT) I'd argue they don't actually exist in the wild. According to common terms, those would be

EVB -> Evaluation board
EVT -> Engineering Validation Testing
DVT -> Design Validation Testing
PVT -> Production Validation Testing

PVT is usually the production device right before mass production, and DVT is usually close to PVT software-wise I believe. In this case I'd guess they decided to not respin stuff after DVT for nd1 but had to have some last minute changes for PVT in ple.

In any case I'd argue those don't exist in the wild and we usually skip prototypes in lk2nd (unless of course you actually have one in your hands)

Reply:
Oh ok, I didn't know the devices were prototypes neither know the terminology, I just assumed they were variants because the ND1 have 4 models (TA-1053, TA-1024, TA-1044, TA-1027) for different markets. In that case it trims down my mainlining process because the EVB for my device has a Synaptics touchscreen but the DVT and my device has a Novatek touchscreen.

In the case of the PLE the change between DVT -> PVT was a single voltage output of one led, probably it was burnt.

Thanks for the clarifications, I will close the PR.